client/src/luigi-client.js

```diff
@@ -443,6 +448,52 @@ const LuigiClient = {
           promises.confirmationModal.rejectFn = reject;
         });
         return promises.confirmationModal.promise;
+      },
+
+      /**
+       * Shows an alert.
+       * @param {Object} settings the settings for the alert
+       * @param {string} settings.text the content of the alert. To add a link to the content, you have to setup the link in `links` object. The key(s) in the `links` object must be used in the text to reference the links, wrapped in curly braces with no spaces. If no text is specified, the alert will not be displayed 
```

Comment:
```suggestion
       * @param {string} settings.text the content of the alert. To add a link to the content, you have to set up the link in the `links` object. The key(s) in the `links` object must be used in the text to reference the links, wrapped in curly brackets with no spaces. If you don't specify any text, the alert is not displayed.
```

I was wondering if we should keep the dot in a parameter ref if we have more sentences to it - usually we don't but maybe it makes sense here.